src/ceph-disk/ceph_disk/main.py

```diff
@@ -3001,10 +3001,19 @@ def start_daemon(
                 ],
             )
         elif os.path.exists(os.path.join(path, 'systemd')):
+            # ensure there is no duplicate ceph-osd@.service
+            command_check_call(
+                [
+                    'systemctl',
+                    'disable',
+                    'ceph-osd@{osd_id}'.format(osd_id=osd_id),
```

Comment:
unless you think that's important and likely to happen, I'd rather have that in a separate commit for cleanup to keep this one minimal. I'm under the impression that systemd will keep that naming convetion to avoid backward incompatible problems.